ts/features/zendesk/screens/ZendeskSeeReportsRouters.tsx

```diff
@@ -89,7 +89,7 @@ const ZendeskSeeReportsRouters = (props: Props) => {
     }
   }, [ticketNumber, dispatch]);
 
-  if (!isStrictSome(ticketNumber) && !pot.isNone(ticketNumber)) {
+  if (pot.isLoading(ticketNumber)) {
```

Comment:
What would happen if this `pot` is in the `error` state? 
From what I can see, `LoadingErrorComponent` could be used to display both `error` and and `loading` states, with `isLoading` prop used to choose which state use, but now is only used fro the `loading` state. 
Am I missing something?

Reply:
Or we could do something like this: 

```suggestion
  if (pot.isLoading(ticketNumber) || pot.isError(ticketNumber)) {
```
in this case `LoadingErrorComponent` is rendered in both cases. What do you think?

Reply:
I think in this scenario this would be enough since there's no reload logic, btw I'm tracking another issue on Jira to suggest the replacement of `pot` with a `RemoteValue`